clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -565,6 +557,22 @@ private Struct fetchResponse(ByteBuffer buffer, short error, long hw, int thrott
         return response.toStruct();
     }
 
+    private MetadataResponse newMetadataResponse(String topic, Errors error) {
+        List<MetadataResponse.PartitionMetadata> partitionsMetadata = new ArrayList<>();
+        if (error == Errors.NONE) {
+            for (PartitionInfo partitionInfo : cluster.partitionsForTopic(topic)) {
+                partitionsMetadata.add(new MetadataResponse.PartitionMetadata(
+                        Errors.NONE, partitionInfo.partition(),
```

Comment:
Nitpick: seems like we are using one parameter per line, so `partitionInfo.partition()` should be on the next line?